templates/snippets/footer.html

```diff
@@ -34,7 +34,6 @@
         </section>
         <div class="contact ui inverted horizontal link list">
             <span class="item">©2018 Trustees of Princeton University</span>
-            <a class="item" href="https://accessibility.princeton.edu/">Accessibility</a>
```

Comment:
why remove this?

Reply:
In the the last review he did for us (on S&co), JJ said we didn't need to link to the accessibility site in our footer, just to the accessibility assistance. Thought I would update here for consistency now that we know (IDK why I did it on the feature branch, though!).

Reply:
oh! good to know